superset/commands/dataset/create.py

```diff
@@ -61,17 +61,16 @@ def run(self) -> Model:
     def validate(self) -> None:
         exceptions: list[ValidationError] = []
         database_id = self._properties["database"]
-        table_name = self._properties["table_name"]
         schema = self._properties.get("schema")
         catalog = self._properties.get("catalog")
         sql = self._properties.get("sql")
         owner_ids: Optional[list[int]] = self._properties.get("owners")
 
-        table = Table(table_name, schema, catalog)
+        table = Table(self._properties["table_name"], schema, catalog)
```

Comment:
I think the previous version is more readable as it groups variables definition in just one place.
```suggestion
        table = Table(table_name, schema, catalog)
```

Reply:
@michael-s-molina personally I try to remove single use variables (where possible) as I find it easier to read/grok the code.